app/models/secret_storage.rb

```diff
@@ -50,6 +50,43 @@ def self.keys
     end
   end
 
+  class HashicorpVault
+
+    SECRET_BACKEND ='secret/'.freeze
```

Comment:
`PREFIX` ... or `FOLDER` ?
also should be `samson/` since everything in vault is secret ?